src/site/topic26.rst

```diff
@@ -5,6 +5,112 @@ Topic #26 --- More Sorting
 * Other pretty cool sorting algorithms
 
 
+Mergesort
+=========
```

Comment:
Imo, missing the recursive definition. I find mergesort much easier to understand from it (yes, _easier_). Something like the following I think explains a lot implicitly.
```java
T[] sort(T[] array) {
    return merge(sort(frontHalfOf(array)), sort(backHalfOf(array)));
}
```

Don't really need to add `merge` or `frontHalfOf` implementations, or the recursive array base cases. I just find actual code to read a lot more clearly than uh, words.

Reply:
I added some stuff now.